app/extend_block.go

```diff
@@ -0,0 +1,27 @@
+package app
+
+import (
+	"github.com/celestiaorg/celestia-app/pkg/da"
+	"github.com/celestiaorg/celestia-app/pkg/shares"
+	"github.com/celestiaorg/celestia-app/pkg/square"
+	"github.com/celestiaorg/rsmt2d"
+	coretypes "github.com/tendermint/tendermint/types"
+)
+
+// ExtendBlock extends the given block data into a data square for a given app
+// version.
+func ExtendBlock(data coretypes.Data, appVersion uint64) (*rsmt2d.ExtendedDataSquare, error) {
+	// Construct the data square from the block's transactions
+	dataSquare, err := square.Construct(data.Txs.ToSliceOfBytes(), appVersion, 64)
```

Comment:
[blocking] I may be viewing an outdated commit but `64` still appears hard-coded

Reply:
gah sorry I reverted to fix different feedback, and it also reverted this change, fixed in https://github.com/celestiaorg/celestia-app/pull/2012/commits/5e7ea5e80f541ee358446435920b4ee1006e196f